src/server/index.js

```diff
@@ -21,3 +22,11 @@ if (module.hot) {
 
   module.hot.accept();
 }
+
+if (__DEV__ && settings.subscription.enabled) {
```

Comment:
Try moving this to module and abstract it similarly as `middlewares` is ding for `app.use`.

Reply:
Agreed, and added.